src/pages/EnablePayments/walletTermsPropTypes.js

```diff
@@ -12,4 +12,7 @@ export default PropTypes.shape({
 
     /** When the user accepts the Wallet's terms in order to pay an IOU, this is the ID of the chatReport the IOU is linked to */
     chatReportID: PropTypes.string,
+
+    /** Boolean to indicate whether the submission of wallet terms is being processed */
+    submitting: PropTypes.bool,
```

Comment:
Very small change, but I think we should call this `isLoading` to follow our naming patterns for bool props